lib/client.js

```diff
@@ -120,17 +126,52 @@ exports.getHistory = async ({
   return res.data;
 };
 
+const _signRequest = async ({keyId, url, json}) => {
```

Comment:
Switch style and move to bottom of file as a helper.

```suggestion
async function _signRequest({keyId, url, json}) {
```